src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/avoidnestedblocks/Example2.java

```diff
@@ -10,7 +10,7 @@
 
 package com.puppycrawl.tools.checkstyle.checks.blocks.avoidnestedblocks;
 
-import static org.checkstyle.suppressionxpathfilter.interfaceistype.SuppressionXpathRegressionInterfaceIsType1.a;
+import static org.checkstyle.suppressionxpathfilter.interfaceistype.InputXpathInterfaceIsType1.a;
 
 // xdoc section -- start
 public class Example2 {
```

Comment:
4. Same as above. Had to change import with new name in avoidnestedblocks/Example2.java